modules/virt-about-storage-volumes-for-vm-disks.adoc

```diff
@@ -0,0 +1,28 @@
+// Module included in the following assemblies:
+//
+// * virt/about-virt.adoc
+
+:_content-type: CONCEPT
+[id="virt-about-storage-volumes-for-vm-disks_{context}"]
+= About storage volumes for virtual machine disks
+
+If you use the storage API with known storage providers, volume and access modes are selected automatically. However, if you use a storage class that does not have a storage profile, you must select the volume and access mode.
+
+For best results, use `accessMode: ReadWriteMany` and `volumeMode: Block`. This is important for the following reasons:
+
+* The ReadWriteMany (RWX) access mode is required for live migration.
+
+* The `Block` volume mode performs significantly better in comparison to the `Filesystem` volume mode. This is because the `Filesystem` volume mode uses more storage layers, including a file system layer and a disk image file. These layers are not necessary for VM disk storage.
+
+For example, if you use {rh-storage-first}, Ceph RBD volumes are preferable to CephFS volumes.
```

Comment:
Suggestion: Align the example with the preceding bullet.

Reply:
It's funny, I had it like that before, but I second guessed it. 😹 Adding the `+` back.